templates/strapdown.html.hbs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<title>API Document</title>
```

Comment:
i'd remove this block; the internal page links (ex. document.html#/definitions/WorkerInfo) navigate the page in a way where the title (ex. WorkerInfo in the above example) is covered up by this title

Reply:
Removed